src/components/segmented-control/segmented-control.tsx

```diff
@@ -57,35 +57,53 @@ export const SegmentedControl = <T extends unknown>({
     [selectedIndex, tileWidth]
   );
 
+  const renderOption = useCallback(
+    (item: T, index: number) => {
+      const disabled = disabledValuesIndices?.includes(index) ?? false;
+
+      return (
+        <TouchableOpacity
+          disabled={disabled}
+          key={index}
+          style={[styles.itemContainer, { width: tileWidth }]}
```

Comment:
tileWidth means titleWidth?

Reply:
No, it's exactly the width of a tile in a switcher

Reply:
oops, sorry :)